src/app/fhir-validator/components/validator-core/validator-core.component.ts

```diff
@@ -40,10 +40,10 @@ export class ValidatorCoreComponent implements OnInit, OnChanges {
   validationErrorStr: string; // We use this value to store preliminary error messages or a generic error message.
   hasResponseData = false;  // Indicates if the response generated any messages. If true, we render the report
   parsedFhirResource: any; // We store value of the validator result in order to present it to the user.
-  displayedColumns: string[] = ['toggle', 'icon', 'severity', 'fhirPath', 'location'];
+  displayedColumns: string[] = ValidatorConstants.displayedColumns;
   isLoading = false;
   allExpanded = true; // Used to render collapsed/expanded all icon as well as calculate if all results are expanded/collapsed
-  severityLevels: string[] = ['error', 'warning', 'information', 'note'];
+  severityLevels: string[] = ValidatorConstants.severityLevels;
```

Comment:
I think it is better if you capitalize the constant names. Something like ValidatorConstants.severityLevels should be ValidatorConstants.SEVERITY_LEVELS. This way we know we refer to constants.